### PR TITLE
Heroku api

### PR DESCRIPTION
**Pull request by Danrod16**

### CODE REVIEW

File: app/services/subdomain_creator.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        body = {
+            "hostname": "#{@subdomain}.avenist.com",
+            "sni_endpoint": nil
+        }
+        body = body.to_json
+        Faraday.post("#{@base_url}/equinox-admin/domains", body, @headers)
+    end
+
+    def subdomain_exists?(data, subdomain)
+        subdomain = false
```
Comment (christinasugiono) [inline app/services/subdomain_creator.rb:42]: do you think this subdomain might get mixed up with the parameter "subdomain"?

File: app/services/subdomain_creator.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        rescue Faraday::ConnectionFailed
+            if (retries += 1) <= 3
+                sleep(retries * 3)
+                retry
+            end
+        end
+    end
+end
+
+SubdomainCreator.new("Helio-digital-studio").call
```
Comment (christinasugiono) [inline app/services/subdomain_creator.rb:71]: This line was for testing right? :D We can remove the line now

Comment (Danrod16) [reply app/services/subdomain_creator.rb:71]: OH yes true, good catch

File: app/services/subdomain_creator.rb (hunk 1 above)
Comment (Danrod16) [reply app/services/subdomain_creator.rb:42]: It is the parameter subdomain being passed int the controller so I think it's okay that it is called the same. 👍

**Pull request merged**